code/ATMOSPHERICS/components/binary_devices/valve.dm

```diff
@@ -218,8 +214,15 @@
 		investigation_log(I_ATMOS,"was [(open ? "opened" : "closed")] by a signal")
 
 /obj/machinery/atmospherics/binary/valve/npc_tamper_act(mob/living/L)
+	toggle_status()
```

Comment:
this will result in an allowed(null) check, make sure that doesn't runtime

Reply:
yeah, these were definitely a mistake